nifi-extension-bundles/nifi-aws-bundle/nifi-aws-processors/src/test/java/org/apache/nifi/processors/aws/kinesis/stream/record/TestAbstractKinesisRecordProcessor.java

```diff
@@ -72,7 +73,7 @@ public void setUp() {
 
         // default test fixture will try operations twice with very little wait in between
         fixture = new AbstractKinesisRecordProcessor(processSessionFactory, runner.getLogger(), "kinesis-test",
-                "endpoint-prefix", null, 10_000L, 1L, 2, DATE_TIME_FORMATTER) {
+                "endpoint-prefix", null, 10_000L, 1L, 2, DATE_TIME_FORMATTER, NoopRecordProcessorBlocker.create()) {
```

Comment:
Can the `StandardRecordProcessorBlocker` be used here? It seems like the Noop instance is not required for standard behavior in this case.